### PR TITLE
feat: add config option for animation transition

### PR DESCRIPTION
**Pull request by aidenybai**
Adds a `animationDuration` option for adjusting the transition time.

### CODE REVIEW

**Review by Just-Moh-it — CHANGES_REQUESTED** [review]
You might wanna add the property to [this README section](https://github.com/harshhhdev/kmenu/blob/master/README.md#customising-the-menu) too, rest lgtm!

File: src/CommandWrapper.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const CommandWrapper: FC<
             initial={{ opacity: 0, y: 20 }}
             animate={{ opacity: 1, y: 0, scale: animate ? 0.98 : 1 }}
             exit={{ opacity: 0, y: 20 }}
-            transition={{ duration: 0.1 }}
+            transition={{ duration: config?.animationDuration || 0.1 }}
```
Comment (Just-Moh-it) [inline src/CommandWrapper.tsx:51]: I think a `??` would be better here for cases where the animation needs to be instant! 🙂

Comment (aidenybai) [reply src/CommandWrapper.tsx]: done!

File: README.md
Comment (Just-Moh-it) [inline README.md:1]: @harshhhdev you might wanna add some linting stuff to your editor... so many inconsistencies lol

**Review by Just-Moh-it — APPROVED** [review]
Great work!

**Comment by aidenybai** [comment]
@harshhhdev pinging here for visibility

**Comment by harshhhdev** [comment]
Hey @aidenybai, thanks for the PR.

It looks good to me—sorry for the delay, I've been busy with some stuff.

**Comment by harshhhdev** [comment]
As stated on that mention, this should be out on v1.3 which I'm aiming to release later today. Thanks for your patience.

**Comment by aidenybai** [comment]
no problem, thanks for the help! we're using it at @dimensionhq for our UI :)

**Pull request merged**